op-e2e/config/init.go

```diff
@@ -287,17 +287,15 @@ func initAllocType(root string, allocType AllocType) {
 				}
 			}
 
-			if allocType == AllocTypeEspressoWithoutEnclave {
-				batcherPk, err := crypto.HexToECDSA(ESPRESSO_PRE_APPROVED_BATCHER_PRIVATE_KEY)
+			// Configure Espresso allocation types
+			if allocType == AllocTypeEspresso || allocType == AllocTypeEspressoWithoutEnclave || allocType == AllocTypeEspressoWithEnclave {
+				batcherPk, err := crypto.HexToECDSA(ESPRESSO_NON_TEE_BATCHER_PRIVATE_KEY)
 				if err != nil {
 					panic(fmt.Errorf("failed to parse batcher private key: %w", err))
 				}
 				intent.Chains[0].EspressoEnabled = true
-				intent.Chains[0].PreApprovedBatcherKey = crypto.PubkeyToAddress(batcherPk.PublicKey)
-			}
-
-			if allocType == AllocTypeEspressoWithEnclave {
-				intent.Chains[0].EspressoEnabled = true
+				intent.Chains[0].NonTeeBatcher = crypto.PubkeyToAddress(batcherPk.PublicKey)
+				intent.Chains[0].TeeBatcher = crypto.PubkeyToAddress(batcherPk.PublicKey)
```

Comment:
Is it for simplicity that we now set `TeeBatcher` even when `allocType == AllocTypeEspressoWithoutEnclave`? I think it's fine to set but not use it--just wanted to make sure it's intentional to put it in this condition clause.